src/libs/antares/study/variable-print-info.h

```diff
@@ -97,30 +94,30 @@ class AllVariablesPrintInfo
 {
 public:
     // Public methods
-    AllVariablesPrintInfo();
-    ~AllVariablesPrintInfo();
+    AllVariablesPrintInfo() = default;
+    ~AllVariablesPrintInfo() = default;
 
-    void add(VariablePrintInfo* v);
+    void add(std::string name, VariablePrintInfo v);
     void clear();
-    VariablePrintInfo* operator[](uint i) const;
+    VariablePrintInfo& operator[](uint i);
     size_t size() const;
-    bool isEmpty() const;
+    bool exists(std::string name);
```

Comment:
Use `const std::string&` or `std::string_view`

Reply:
I tried to replace **std::string** by **std::string_view** or  **const std::string&** in arguments of member functions : 
- **add(...)**
- **exists(...)**
- **setPrintStatus(...)**
- **isPrinted(...)**

but they all call **to_uppercase()** internally, that makes changes on the **std::string**.
So I leave the code as it is regarding this.